api/types.proto

```diff
@@ -359,6 +359,15 @@ message TaskStatus {
 	}
 }
 
+message AttachmentStatus {
```

Comment:
I can add a new enum type, just that I won't be able to call any of its values "ALLOCATED", given that is already used for TaskStatus enum (code generator complains for that). I can get around with this via "RESERVED".  Will make the change.